test/lib/mayaUsd/fileio/testPrimReader.py

```diff
@@ -33,24 +33,18 @@ def Read(self, context):
         sphere = UsdGeom.Sphere(usdPrim)
         cmds.polySphere(r=sphere.GetRadiusAttr().Get(), name=usdPrim.GetName())
         MayaNode = context.GetMayaNode(usdPrim.GetPath().GetParentPath(), True)
-        print("primReaderTest.Read Ok")
```

Comment:
Unrelated changes, apologies, could not help cleaning this up while looking at files in this directory.